relayer/config/config.go

```diff
@@ -67,6 +68,9 @@ type Config struct {
 	DeciderURL             string                   `mapstructure:"decider-url" json:"decider-url"`
 	SignatureCacheSize     uint64                   `mapstructure:"signature-cache-size" json:"signature-cache-size"`
 
+	// mapstructure doesn't handle time.Time out of the box so handle it manually
+	EtnaTime time.Time `json:"etna-time"`
```

Comment:
I would have liked to do `omitempty` so that tests don't write the field out when marshaling JSON here but zero time value still gets written out unless the field type is `*time.Time` which seemed messier.

The value that does get written out is correctly parsed by viper to be zero time value.  As a result tests run the pre-Etna case unless otherwise specified.